### PR TITLE
OSDOCS#6505: Alibaba - move install config parameters out of assemblies

### PR DESCRIPTION
**Pull request by mjpytlak**
Version(s):
4.14+

Issue:
This PR addresses [osdocs-6505](https://issues.redhat.com/browse/OSDOCS-6505).

Link to docs preview:

- [Installation configuration parameters for Alibaba Cloud](https://64049--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_alibaba/installation-config-parameters-alibaba)
  New assembly at book level.
- Installing a cluster on Alibaba Cloud with customizations > [Creating the installation configuration file](https://64049--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_alibaba/installing-alibaba-customizations#installation-initializing_installing-alibaba-customizations)
Note the absence of Installation configuration parameters following this topic. In lieu of the module, a cross reference is provided for the new assembly.

It is worth noting that the latter is an example of the change that this PR introduces. This change has been applied to all Alibaba installation assemblies.

QE review:
- [NA] QE has approved this change.
Given that this update is a reorg only, QE approval is not required. See conversation at https://github.com/openshift/openshift-docs/pull/61165#pullrequestreview-1493117699

### CODE REVIEW

File: installing/installing_alibaba/installation-config-parameters-alibaba.adoc [hunk 1]
```diff
@@ -0,0 +1,12 @@
+:_content-type: ASSEMBLY
+[id="installation-config-parameters-alibaba"]
+= Installation configuration parameters for Alibaba Cloud
+include::_attributes/common-attributes.adoc[]
+:context: installation-config-parameters-alibaba
+:platform: Alibaba Cloud
+
+toc::[]
+
+Before you deploy an {product-title} cluster on Alibaba Cloud, you provide parameters to customize your cluster and the platform that hosts it. When you create the `install-config.yaml` file, you provide values for the required parameters through the command line. You can then modify the `install-config.yaml` file to customize your cluster further.
```
Comment (opayne1) [inline installing/installing_alibaba/installation-config-parameters-alibaba.adoc:10]: This is just a suggestion, but I almost feel like there should be a word after you and provide in this paragraph. The [style guide](https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#second-person) has something similar listed in one of the examples. 

```suggestion
Before you deploy an {product-title} cluster on Alibaba Cloud, you must provide parameters to customize your cluster and the platform that hosts it. When you create the `install-config.yaml` file, you must provide values for the required parameters through the command line. You can then modify the `install-config.yaml` file to customize your cluster further.
```

**Review by opayne1 — COMMENTED** [review]
I had one suggestion but otherwise it looks good to me!

**Comment by sjstout** [comment]
LGTM

File: installing/installing_alibaba/installation-config-parameters-alibaba.adoc (hunk 1 above)
Comment (JoeAldinger) [reply installing/installing_alibaba/installation-config-parameters-alibaba.adoc:10]: I'd say if not `must` maybe `can`.

**Pull request merged**